pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: very_good_cli
 description: A Very Good Command Line Interface for Dart created by Very Good Ventures.
 version: 0.6.9
 homepage: https://github.com/VeryGoodOpenSource/very_good_cli
+publish_to: none
```

Comment:
Is this just for testing?

Reply:
yeah since I'm pointing to the very_good_test_runner via git url temporarily. Looks like the pipeline is all green so I'll publish `very_good_test_runner` and update the PR to reference the published version.